src/script/lua_api/l_mapgen.cpp

```diff
@@ -476,6 +476,29 @@ size_t get_biome_list(lua_State *L, int index,
 
 ///////////////////////////////////////////////////////////////////////////////
 
+// get_containing_chunk(blockpos, [chunksize])
+int ModApiMapgen::l_get_containing_chunk(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	v3s16 blockpos = check_v3s16(L, 1);
+
+	s16 chunksize;
+	if (lua_isnumber(L, 2)) {
+		chunksize = luaL_checkinteger(L, 2);
+	} else {
+		MapSettingsManager *settingsmgr =
+			getServer(L)->getEmergeManager()->map_settings_mgr;
+		std::string chunksize_str = "5";
```

Comment:
hm, isn't this already assured to be set by some code that runs prior?

Reply:
I'll just change the default value to the empty string.